Splunk docs CIM error added (#942)
demisto/content-docs: content-repo/extra-docs/articles/splunk-add-on.md

* In the case of a self-signed certificate, make sure that the whole certificate chain exist. Each of root, intermediate, and client certificate is required.

* When checking SSL certificate validity by pinging, ping to the server domain. SSL certificates are signed on server domain rather than its IP.

> #### Splunk Common Information Model (CIM) issues
> * If Splunk CIM is installed make sure it's version match the installed Splunk version. 
> * A common error caused by incompatible version might look as the following:

Reviewer: ```suggestion
#### Splunk Common Information Model (CIM) Issues
If Splunk CIM is installed, make sure its version is compatible with the installed Splunk version. A common error caused by a version incompatibility might look as follows:
```
